rpc_jobs/standard_job_premerge.yml

```diff
@@ -1,23 +1,3 @@
-- project:
-    name: "gating-pre-merge"
-    repo_name: "rpc-gating"
-    repo_url: "https://github.com/rcbops/rpc-gating"
-    # `series` defaults to `master` and can be omitted.  Note that `series` is
-    # only used in the job name and does not necessarily need to map to a
-    # branch name (thought in most cases it will).
-    #series: "master"
-    branches:
-      - "master"
-    image:
-      - "xenial":
-          SLAVE_TYPE: "nodepool-ubuntu-xenial-g1-8"
-    scenario:
-      - "lint"
-    action:
-      - "test"
-    jobs:
-      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
-
```

Comment:
Is this definitely no longer needed? I'm not seeing a lint job under the unit tests.

Reply:
Yeah, OK - I've restored this job. Good catch!